appinterface/projection/crossfire/crossfire.go

```diff
@@ -115,14 +114,34 @@ func (projection *Crossfire) HandleEvents(height int64, events []event_entity.Ev
 		return fmt.Errorf("error projecting validator view: %v", err)
 	}
 
-	// Recompute TxSentRank
-	errTxSentRank := projection.computeTxSentRank(crossfireValidatorsStatsView, crossfireValidatorsView)
-	if errTxSentRank != nil {
-		return fmt.Errorf("[error] Updating TxSentTask Rank %w", errTxSentRank)
+	// Ranks computation at the end of event projection
+	if blockTime.Before(projection.competitionEndTime) {
```

Comment:
Explanation: add endtime check so that the tx_rank will be frozen after crossfire ended